docs/source/configuration.rst

```diff
@@ -0,0 +1,89 @@
+.. role:: python(code)
```

Comment:
These files have the same content as the READMEs right? Is it possible to generate the rst files from the markdown? Otherwise, documentation always has to be updated in two places.

Reply:
- discussed in the group: Have the Wiki for tutorials separate. The readme should mainly contain our USPs.